packages/flutter_tools/test/general.shard/android/gradle_test.dart

```diff
@@ -98,33 +98,33 @@ void main() {
   group('gradle tasks', () {
     test('assemble release', () {
       expect(
-        getAssembleTaskFor(const BuildInfo(BuildMode.release, null)),
+        getAssembleTaskFor(const BuildInfo(BuildMode.release, null, fontSubset: false)),
```

Comment:
nit: `fontSubset` default could be `false`.

Reply:
I didn't want to miss any invocations of it anywhere.  I think there's also a slight preference for requiring parameters to be set in the tool so that we avoid surprise defaults